ui/src/components/home/HomeContainer.js

```diff
@@ -38,9 +39,10 @@ const HomeContainer = (props) => {
   };
   return (
     <Flex direction="column">
-      <HeroSection />
-      <EventsSection />
+      <HeroSection sermonRef={sermonRef} />
```

Comment:
why is this called sermonRef??

Reply:
because it directs to the hello & sermons section, also not sure what else to name it lol